app/views/orders/show.html.erb

```diff
@@ -11,18 +11,18 @@
       </tr>
     </thead>
     <tbody>
-      <% @order.items.each do |item| %>
+      <% @order.order_items.each do |order_item| %>
       <tr>
-        <td><%= link_to item.title, item_path(item) %></td>
-        <td><%= number_to_currency(item.price, precision: 2) %></td>
-        <td><%= item.get_quantity_in_order(@order)%></td>
-        <td><%= number_to_currency(item.get_subtotal_on_order(@order), precision: 2) %></td>
+        <td><%= link_to order_item.item.title, item_path(order_item.item) %></td>
```

Comment:
I like this. We should look into if the methods .get_subtotal_on_item and .get_quantity_in_order are used anywhere else, if not we can take them out.